src/shipping/consignment-action-creator.ts

```diff
@@ -301,8 +301,14 @@ export default class ConsignmentActionCreator {
                 .map(item => ({
                     itemId: item.id,
                     quantity: item.quantity,
-                })
-            ),
+                }))
+                .concat(
+                    (cart.lineItems && cart.lineItems.customItems || [])
+                        .map(item => ({
```

Comment:
Instead of duplicating the `map` function, would it be better if you just concat the `physicalItems` and `customItems` before `map`? i.e.:

```ts
const { physicalItems, customItems = [] } = cart.lineItems;

[...physicalItems, ...customItems].map(item => ({
    itemId: item.id,
    quantity: item.quantity,
}));
```

Reply:
Thanks, that worked! I tried using `physicalItems.concat()` but it was complaining that the types were different.

Reply:
@davidchin  let me know if it's good to go